x/crosschain/simulation/operations_test.go

```diff
@@ -0,0 +1,23 @@
+package simulation_test
+
+import (
+	"math"
+	"math/rand"
+	"testing"
+
+	"github.com/zeta-chain/node/x/crosschain/simulation"
+)
+
+func Test_Matrix(t *testing.T) {
+	r := rand.New(rand.NewSource(42))
+	numVotes := 10
+	for i := 0; i < 10; i++ {
+
+		ballotVotesTransitionMatrix, yesVotePercentageArray, ballotVotesState := simulation.BallotVoteSimulationMatrix()
```

Comment:
What does this test test?

Reply:
removing it,it was just some debugging code